### ISSUE TITLE
Nest-Cam

### ISSUE THREAD

**Issue by MalcolmRussell** [hoobs-org/HOOBS#1431]
**Description**

After upgrading to Hoobs V4 Nest-Cam certified plugin has failed.  When the bridge starts the plugin fails to load. I have unistalled and reinstalled and configured many times but get the same issue every time(see log bellow)

**Expected behavoir**

Plugin should load when bridge started

**Plugin name**

Nest-Cam

**Please report the issue to the plugin developer**

https://github.com/Brandawg93/homebridge-nest-cam/issues/428

**Version**

4.0.98

**Did you upgrade**

Issue since upgrading to V4

**Pervious version**

3.?

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**

No

**What device are you using?**

raspberry pi 4B  4G

**If this is a custom install, what's your operating system?**

PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

**Post your config**

```
{
    "accessories": [],
    "platforms": [
        {
            "platform": "Nest-cam",
            "refreshToken": "************************",
            "options": {
                "ffmpegCodec": "libx264",
                "alertCheckRate": 10,
                "alertCooldownRate": 180
            }
        }
    ]
}
```

**Post your log**

```
30/07/2021, 16:36:45Nest Cam BridgeInitializing HAP-NodeJS v0.9.4...
30/07/2021, 16:36:45Nest Cam BridgeERRORError loading plugin "homebridge-nest-cam"
30/07/2021, 16:36:45Nest Cam BridgeWARNINGNo plugins installed.
30/07/2021, 16:36:45Nest Cam BridgeLoading 1 platforms...
30/07/2021, 16:36:45Nest Cam BridgePreparing Advertiser for 'Nest Cam Bridge 1033' using bonjour-hap backend!
30/07/2021, 16:36:45Nest Cam BridgeSetup URI 'X-HM://0023ISYWYULX0'
30/07/2021, 16:36:45Nest Cam BridgeStarting to advertise 'Nest Cam Bridge 1033' using bonjour-hap backend!
30/07/2021, 16:36:45Nest Cam BridgeBridge is running on port 51826.
```

**Comment by Tufail-Ahmed** [comment]
Has there been any update on this?, I'm having the same issue trying to get my nest doorbell cam to connect. I have a very similar setup and I am using the latest version of Hoobs. Previously, I had to supply Token, Cookies and an Api key using an old plugin (no longer supported). The new setup is only requesting a token.

**Comment by askovi** [comment]
Reinstall the nest-cam plugin. It has a visual configuration option now if you click the Link Account button (wait a few seconds for the button to appear on the configuration page after you installed the plugin)

**Comment by MalcolmRussell** [comment]
This does not solve issue for me, visual configuration works as expected but log still shows 'No plugins installed' when the bridge restarts.

**Comment by Tufail-Ahmed** [comment]
Thanks for getting back to me, yes it would appear there is now a Link Account button, which was not previously available to me. I click on on and I get this...
![2021-08-05 18 47 02](https://user-images.githubusercontent.com/47148780/128397434-ebafa9f5-63dd-4a31-838f-660fd9d6375c.jpeg)

but I don't see any login in screen or anywhere to enter my credentials.

**Comment by MalcolmRussell** [comment]
That screen suggest your all ready logged into google. So just copy code and paste into config

**Comment by Tufail-Ahmed** [comment]
I wish Malcolm, it's just an example image. I checked to see if there was a pop being blocked, and I also tried different browsers, turned off ad blockers etc... but nothing. I looked to see if there was a newer version of Hoobs, and there was an update (Version: 4.0.98).

I installed the update and this time, when I clicked on "Login" at the bottom it took me to the login page (YES!). I filled in my credentials, got the code, and pasted in the code here...
![2021-08-05 19 24 13](https://user-images.githubusercontent.com/47148780/128401809-007f969c-7751-48ba-a0ac-21b0fdfbd6c8.jpeg)

It's now working. Try updating the version of Hoobs to the latest version and re-installing the plugin. Let me know how you get on.
